baselines/acktr/run_atari.py

```diff
@@ -1,23 +1,34 @@
 #!/usr/bin/env python3
 
-from functools import partial
-
 from baselines import logger
 from baselines.acktr.acktr_disc import learn
 from baselines.common.cmd_util import make_atari_env, atari_arg_parser
 from baselines.common.vec_env.vec_frame_stack import VecFrameStack
-from baselines.ppo2.policies import CnnPolicy
+from baselines.a2c.policies import CnnPolicy
+
 
 def train(env_id, num_timesteps, seed, num_cpu):
+    """
+    train an ACKTR model on atari
+
+    :param env_id: (str) Environment ID
+    :param num_timesteps: (int) The total number of samples
+    :param seed: (int) The initial seed for training
+    :param num_cpu: (int) The number of cpu to train on
+    """
     env = VecFrameStack(make_atari_env(env_id, num_cpu, seed), 4)
-    policy_fn = partial(CnnPolicy, one_dim_bias=True)
```

Comment:
 @hill-a  one_dim_bias does not exist, no ?

Reply:
it is used for the conv layers of the CNN policies

https://github.com/hill-a/stable-baselines/blob/fixes_cleanup/baselines/a2c/utils.py#L95

Reply:
So you removed it because it was buggy ?

Reply:
good point, although I think it is broken on this branch, I can add it to the refactoring branch